wp-resources/mu-plugins/studio-disable-auto-updates.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Could we keep it as a mu-plugin mounted by Playground and not added to the user's site filesystem?

Also, is there a reason to disable all updates? As we add that plugin depending on the WordPress version selected, it would make sense to disable only auto updates of the WP core, letting users still benefit from automatic updates of plugins or themes.

Could we achieve this by setting the file with the constant? Or just defining constant for Playground environment?

```
define( 'WP_AUTO_UPDATE_CORE', true );
```

Reply:
Here is were we usually set mu-plugins.
https://github.com/Automattic/studio/blob/f0efa981a196265b5d31d058e19bed5496a122b0/vendor/wp-now/src/wp-now.ts#L457-L468

We have some code that already sets that constant in. We should check the current behaviour:
https://github.com/Automattic/studio/blob/f0efa981a196265b5d31d058e19bed5496a122b0/vendor/wp-now/src/wp-now.ts#L427-L429